docker/README.md

```diff
@@ -1,6 +1,70 @@
-## Instructions
+# Running a node with Docker
 
----
-1) Build the image from docker image with `docker build --build-arg version=$(git describe --tags) --build-arg gitCommit=$(git rev-parse HEAD) -t drandorg/go-drand:latest .` in the project root folder
-2) Run `docker-compose -f "docker-compose.yml" up`
-3) Execute the script called `startBeacon.sh` in order to run the setup phase. After that, the nodes will start to generate randomness
+## Prerequisites
+- install docker
+- install docker compose (probably bundled with docker these days)
+- run on linux :) (other platforms may work but YMMV)
+
+## Creation of a keypair
+
+Pull the latest drand image:
+```shell
+docker pull drandorg/go-drand:v1.5.3
```

Comment:
Does this need to be updated to point to the new github registry?

Reply:
Will raise another PR once we get a tagged version pushed